projects/swimlane/ngx-ui/src/lib/components/tip/tip.component.ts

```diff
@@ -36,4 +40,13 @@ export class TipComponent {
   onClose() {
     this.close.emit();
   }
+
+  private getIcon(status: TipStatus): string {
+    const icons = {
```

Comment:
Could we move this outside of the component class?

Reply:
Yes, good catch. Thanks! Done